src/components/DateTimePicker.js

```diff
@@ -145,87 +148,118 @@ const DatePicker = React.createClass({
+              ref='dateSelect'
+              style={Object.assign({}, styles.selectWrapper, this.state.showCalendar ? styles.activeSelectWrapper : null)}
+              tabIndex={0}
+            >
+              {this.props.showIcons ? (
+                <Icon
+                  size={20}
+                  style={styles.selectedIcon}
+                  type={this.props.dateIcon}
+                />
+              ) : null}
+              <div style={styles.selectedText}>
+                {this.props.selectedDate ? moment.unix(this.props.selectedDate).format(this.props.dateFormat) : this.props.datePlaceholder}
+              </div>
+              <Icon
+                size={20}
+                style={styles.selectedDateCaret}
+                type={this.state.showCalendar ? 'caret-up' : 'caret-down'}
+              />
+            {this.state.showCalendar ? (
+              <Calendar
+                {...this.props}
+                onDateSelect={this._handleDateSelect}
+                style={styles.calendar}
+              />
+            ) : null}
             </div>
+          </Column>
+          {this.props.children ? (
+            <Column span={spans.childrenColumn}>
+              <div style={styles.children}>
+                {this.props.children}
+              </div>
+            </Column>
           ) : null}
-        </div>
-      </div>
+          <Column span={spans.timeColumn}>
+            <div
+              onBlur={this._handleTimeBlur}
+              onFocus={this._handleTimeFocus}
+              style={Object.assign({}, styles.selectWrapper, this.state.editTime ? styles.activeSelectWrapper : null)}
+              tabIndex={0}
+            >
+              {this.props.showIcons ? (
+                <Icon
+                  size={20}
+                  style={styles.selectedIcon}
+                  type={this.props.timeIcon}
+                />
+              ) : null}
+              {this.state.editTime ? (
+                <input
+                  autoFocus={true}
+                  defaultValue={this.state.editTime ? moment.unix(this.props.selectedDate).format('HH:mm') : null}
+                  name='time'
+                  onBlur={this._handleTimeSelect}
+                  ref='timeInput'
+                  style={styles.timeInput}
+                  type='time'
+                />
+              ) : (
+                <div style={styles.timeDisplay}>
+                  {this.props.selectedDate ? moment.unix(this.props.selectedDate).format(this.props.timeFormat) : this.props.timePlaceholder}
+                </div>
+              )}
+              {this.props.timezoneFormat ? (
+                <div style={styles.timezone}>
+                  {this._getTimezone(this.props.selectedDate)}
+                </div>
+              ) : null}
+            </div>
+          </Column>
+        </Row>
+      </Container>
     );
   },
 
+  spans () {
+    return Object.assign({}, {
```

Comment:
Why the use of `Object.assign` here?  It doesn't look like your merging with anything other than just an empty object.